blazor/datagrid/column-menu.md

```diff
@@ -90,45 +122,284 @@ In the following sample, Order ID, Order Date, and Freight columns are sorted at
+
+<SfGrid DataSource="@Orders" AllowPaging="true" ShowColumnMenu="true">
+    <GridPageSettings PageSize="8"></GridPageSettings>
+    <GridColumns>
+        <GridColumn Field=@nameof(OrderData.OrderID) HeaderText="Order ID" TextAlign="TextAlign.Center" Width="120"></GridColumn>
+        <GridColumn Field=@nameof(OrderData.CustomerID) HeaderText="Customer Name" TextAlign="TextAlign.Center" Width="120"></GridColumn>
+        <GridColumn Field=@nameof(OrderData.Freight) HeaderText="Freight" Format="C2" TextAlign="TextAlign.Center" Width="120"></GridColumn>
+        <GridColumn Field=@nameof(OrderData.ShipName) HeaderText="Ship Name" TextAlign="TextAlign.Center" Width="120"></GridColumn>
+    </GridColumns>
+</SfGrid>
+<style>
+    .e-grid .e-columnheader .e-icons.e-columnmenu::before {
+        .content: "\e84f";
+    }
+</style>
+@code {
+ 
+    public List<OrderData> Orders { get; set; }   
+    protected override void OnInitialized()
+    {
+        Orders = OrderData.GetAllRecords();
+    }
+}
+{% endhighlight %}
+{% highlight c# tabtitle="OrderData.cs" %}
+    public class OrderData
+    {
+        public static List<OrderData> Orders = new List<OrderData>();
+        
+       
+        public OrderData()
+        {
+
+        }
+        public OrderData(int? OrderID, string CustomerID, double Freight,string ShipCity)
+        {
+            this.OrderID = OrderID;
+            this.CustomerID = CustomerID;
+            this.Freight = Freight;
+            this.ShipName = ShipCity;         
+
+        }
+        public static List<OrderData> GetAllRecords()
+        {
+            if (Orders.Count() == 0)
+            {
+                int code = 10;
+                for (int i = 1; i < 2; i++)
+                {
+                    Orders.Add(new OrderData(10248, "ALFKI",33.32, "France"));
+                    Orders.Add(new OrderData(10249, "ANANTR", 34.32, "Germany"));
+                    Orders.Add(new OrderData(10250, "ANTON", 36.32, "Brazil"));
+                    Orders.Add(new OrderData(10251, "BLONP", 54.31, "Belgium"));
+                    Orders.Add(new OrderData(10252, "BOLID", 35.36, "Switzerland"));
+                    Orders.Add(new OrderData(10253, "ANTON", 37.35, "Switzerland"));
+                    Orders.Add(new OrderData(10254, "BLONP", 33.32, "Germany"));
+                    Orders.Add(new OrderData(10255, "BOLID", 76.74, "Germany"));
+                    Orders.Add(new OrderData(10256, "ALFKI",55.43, "Belgium"));                   
+                    code += 5;
+                }
+            }
+            return Orders;
+        }
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public double? Freight { get; set; }
+        public string ShipName { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "https://blazorplayground.syncfusion.com/embed/VtrAWsDhqDmEYwkF?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
```

Comment:
changes are not reflected in the sample.

Reply:
modified.